src/app/Upload.svelte

```diff
@@ -16,17 +29,16 @@
         if (item.kind === "file") {
```

Comment:
**[Suggestion]**: This nested logic is not very readable and can be done by using a `&` logic. I would also use ternary operator to further clean up